Framework/PythonInterface/test/python/plugins/algorithms/IndirectTransmissionTest.py

```diff
@@ -123,12 +123,12 @@ def test_mass_density(self):
         ws = IndirectTransmission(Instrument=instrument, Analyser=analyser, Reflection=reflection,
                                   ChemicalFormula=formula, DensityType='Mass Density', Density=density, Thickness=thickness)
 
-        ref_result = [6.65800, 0.82122, 2.58186, 53.50693, 56.08880, 0.03342, 0.1, 0.82676, 0.17096]
+        ref_result = [6.65800, 0.82122, 2.58186, 53.50693, 56.08880, 0.100272, 0.1, 0.56512, 0.43021]
         mass_values = ws.column(1)
         np.testing.assert_array_almost_equal(mass_values, ref_result, decimal=4)
 
         # mass density 1.0 = number density 0.033424 for water
```

Comment:
The comment and code here are now inconsistent. I would suggest getting rid of the comment, it doesn't seem that useful.